subsys/bluetooth/host/sdp.c

```diff
@@ -29,8 +29,6 @@
 
 #define SDP_CHAN(_ch) CONTAINER_OF(_ch, struct bt_sdp, chan.chan)
 
-#define IN_RANGE(val, min, max) (val >= min && val <= max)
-
```

Comment:
Shouldn't this file get an explicit `#include <sys/util.h>`?

Reply:
I can do that, but sdp.c includes `#include <bluetooth/buf.h>`, which in term includes `#include <sys/util.h>`. Not sure what the guidelines are here, but if it is that we should include all header files we use explicitly, we are missing quite a lot of `#include` overall.